### ISSUE TITLE
Question about MAtmos resource packs for 1.7.10

### ISSUE THREAD

**Issue by xJon** [Sollace/MAtmos#17]
The MSI resource pack for 1.7.10 in [the repo's wiki](https://github.com/Sollace/MAtmos/wiki/SoundPack:-MSI) has a broken download link, but online I have been able to find another download through this link: https://ha3.eu/x/tempshare/misc/msi_conversion.zip
Do you think it's the same?

Also, what's the different between this resource pack, the one Hurricaaane posted in the [Minecraft Forums](https://www.minecraftforum.net/forums/mapping-and-modding-java-edition/minecraft-mods/1274487-matmos-environmental-sound-atmosphere-simulator?page=137#c3901) and the new resource packs (MSI + Matmos.Default) [included by you](https://github.com/Sollace/MAtmos/releases/tag/r29c) for 1.8.9+?
I was looking for the original sounds for MAtmos with 1.7.10 and thus I'm not asking about Rhapsodia which still works for 1.7.10.

Thank you!

**Comment by Sollace** [comment]
Matmos Default is just the default matmos sounds. That, IIRC is the sounds Huricaaan posted on the forums.

All the MSI ones are just different versions of MSI updated to their respective minecraft versions.

I'll update the wiki links to include

**Comment by xJon** [comment]
Can the MAtmos Default pack you included in 1.8.9+ be used in 1.7.10? Or is it version-specific like the MSI pack?
Also, it's 5.89 MB and the sounds Huricaaan posted in the forums are 11.6 MB, so I'm not sure what should I use.
Thank you!

**Comment by Sollace** [comment]
I'm not entirely sure myself why they're so different. IIRC a lot of the default resources (for both this and PF) are lost now because @hurricaaane never published them with his mods' source codes.

The default I provide was a best attempt and reconstructing the old files. I can't even remember where the audio files for either project are any more!

**Comment by xJon** [comment]
Damn, that's a shame. And can they not be extracted out of older versions?
Also, can the default you provided be used on MAtmos 1.7.10?

**Comment by DrakoAlcarus** [comment]
There was a version of Default I had found that had all the sounds, just
that they didn't play right. I'll have to find it for you when I get to my
PC. Maybe we can get that to work right when the mod gets updated.

**Comment by Sollace** [comment]
@xJon The default I provided should work with 1.7.10, yes. All I've done is change the mc version so the game doesn't warn you about using them.

**Comment by DrakoAlcarus** [comment]
It seems i never provided which sounds were messing up. In Matmos Default, the high altitude strong wind can be heard everywhere where normal wind should be. The tree wind sound does come through, but the loud wind is always present.

**Comment by Sollace** [comment]
That's a consistent problem with Matmos. It always things there is a strong wind even if there are just a few air blocks around the player.

This is something I've never been able to fix and part of why I've since decided to retire the mod. It just can't be maintained any further.

**Comment by DrakoAlcarus** [comment]
I did make a small mistake. The wind_hit sound was louder than its supposed
to be, but I adjusted their volumes in a copy, and it's more bearable. The
sound issue is the fact that there is no high altitude wind at all.

Tbh all I really want are the classic Tekkit cave sounds. Something about
that deep rumble and cave roars are incredible. I could just download
Tekkit, but I don't know where to find it.
